test/src/unit-tile-metadata.cc

```diff
@@ -787,7 +787,8 @@ struct CPPVarTileMetadataFx {
       // Copy the string.
       a_offsets[i] = offset;
       auto idx = values[i];
-      memcpy(&a_var[offset], strings_[idx].c_str(), strings_[idx].size());
+      memcpy(
+          a_var.data() + offset, strings_[idx].c_str(), strings_[idx].size());
```

Comment:
Same issue as below. Is this check failing?

Reply:
The problem here is the last string of a buffer is 0 sized, we will access the byte next to the end of the buffer, but we won't copy anything to it since the size is 0.